src/main/java/com/thebois/models/inventory/IInventory.java

```diff
@@ -77,4 +91,13 @@
      */
     int numberOf(ItemType itemType);
 
+    /**
+     * Calculates and returns a list of the differing IItems between two inventories.
+     *
+     * @param inventory The inventory to check against.
+     *
+     * @return A list of differing items.
```

Comment:
ItemTypes (good) but the javadoc does not say that. Reading the javadoc I would have thought it would return IItems, might just be me though :p

Reply:
👍